plugins/InspectionGadgets/InspectionGadgetsAnalysis/resources/messages/InspectionGadgetsBundle.properties

```diff
@@ -1657,6 +1657,9 @@ if.may.be.factorized.problem.descriptor=<code>#ref</code> can be factorized #loc
 if.may.be.factorized.quickfix=Replace with factorized expression
 redundant.string.format.call.display.name=Redundant call to 'String.format()'
 redundant.call.problem.descriptor=Redundant call to <code>#ref()</code> #loc
+inspection.simplifiable.compare.java.time.display.name=Expression with 'java.time' 'compareTo' call can be simplified
+inspection.simplifiable.compare.java.time.family.name=Simplify expression with 'java.time' 'compareTo' call
```

Comment:
Please add parentheses to `compareTo` here as well:
`Expression with 'java.time' 'compareTo()' call can be simplified`

Reply:
Fixed, thank you!